WFLY-6459_EAP7-838_JAX-RS_2.1.adoc

```diff
@@ -0,0 +1,60 @@
+= WFLY-6459 / EAP7-838 JAX-RS 2.1 Support
+:author:            Alessio Soldano
+:email:             asoldano@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+This issue has two parts.
+Firstly it brings JAX-RS 2.1 into Wildfly.
+Secondly, it introduces support for switch between JAX-RS 2.0 (EE7) and JAX-RS 2.0 (EE 8) for the sake of certification.
```

Comment:
Typo, there should probably be "JAX-RS 2.0 (EE7) and JAX-RS 2.1 (EE 8)"

Reply:
Fixed, thanks